libvast/include/vast/fwd.hpp

```diff
@@ -407,6 +408,7 @@ CAF_BEGIN_TYPE_ID_BLOCK(vast_types, first_vast_type_id)
   VAST_ADD_TYPE_ID((vast::uuid))
   VAST_ADD_TYPE_ID((vast::table_slice_column))
   VAST_ADD_TYPE_ID((vast::pipeline_ptr))
+  VAST_ADD_TYPE_ID((std::shared_ptr<vast::pipeline>))
```

Comment:
Is this a short-term workaround until the inspection changes are in and pipelines are inpsectable (without the shared_ptr wrapping)?

Reply:
I tried to preserve the original interface (using `shared_ptr<legacy_pipeline>`) as far as possible, but given your comment, there presumably is no reason for `shared_ptr`. I now copied the changes from the other branch and removed the `shared_ptr` with 44a8eaa.

Reply:
Perfect, that's a lot better!